### PR TITLE
Update face-detect.py

### LINKED ISSUE #668: Modify Face Detection
I want to add a few lines of code in face detection which will count the number of faces 
please assign me for this issue
I am a gssoc'21 participant and wants to contribute on this issue

### PR DESCRIPTION
**Pull request by Avishake007**
# Description

I have made some updates in the face-detect.py file:
- Counted number of faces visible
- Added press q to quit

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by santushtisharma10 — CHANGES_REQUESTED** [review]
@Avishake007 

- add a readme file.
- Add `cv2.destroyAllWindows()` at the end of your code

**Comment by Avishake007** [comment]
@santushtisharma10  its done

File: Face-Detection/face-detect.py [hunk 1]
```diff
@@ -12,16 +12,23 @@ def detect():
         gray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
 
         face = face_cascade.detectMultiScale(gray, 1.1, 4)
+        a=str(len(face))
```
Comment (antrikshmisri) [inline Face-Detection/face-detect.py:15]:
```suggestion
        a=len(face)
```

File: Face-Detection/face-detect.py [hunk 2]
```diff
@@ -12,16 +12,23 @@ def detect():
...
+        font = cv2.FONT_HERSHEY_SIMPLEX
+        if int(a)==1:
```
Comment (antrikshmisri) [inline Face-Detection/face-detect.py:17]:
```suggestion
        if a==1:
```

File: Face-Detection/face-detect.py [hunk 3]
```diff
@@ -12,16 +12,23 @@ def detect():
...
+            cv2.putText(img,a+"face detected",(15,350), font, 2,(255,0,0),2,cv2.LINE_AA)
```
Comment (antrikshmisri) [inline Face-Detection/face-detect.py:18]:
```suggestion
            cv2.putText(img,f"{a}face detected",(15,350), font, 2,(255,0,0),2,cv2.LINE_AA)
```

File: Face-Detection/face-detect.py [hunk 4]
```diff
@@ -12,16 +12,23 @@ def detect():
...
+        else:
+            cv2.putText(img,a+"faces detected",(15,350), font, 2,(255,0,0),2,cv2.LINE_AA)
```
Comment (antrikshmisri) [inline Face-Detection/face-detect.py:20]:
```suggestion
            cv2.putText(img,f"{a}face detected",(15,350), font, 2,(255,0,0),2,cv2.LINE_AA)
```

**Review by antrikshmisri — CHANGES_REQUESTED** [review]
No need to type cast `a` , you can simply do this

**Review by kaustubhgupta — COMMENTED** [review]
Wait, you don't need to make changes to the current file. Create a folder called v3 and name the script as face_detectv3.py and put the code there. This is because, in another PR #682, he will also do the same thing as v2. Shift the readme and model XML to v3 folder

**Comment by Avishake007** [comment]
@kaustubhgupta Will I create  the v3 folder inside the face detection folder or in the main folder?

**Comment by kaustubhgupta** [comment]
> @kaustubhgupta Will I create the v3 folder inside the face detection folder or in the main folder?

Inside face detection folder

**Comment by Avishake007** [comment]
@antrikshmisri  and @kaustubhgupta  I have made the necessary changes. Kindly check

File: Face-Detection/v3/face_detectv3.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import cv2
+
+face_cascade = cv2.CascadeClassifier("haarcascade_frontalface.xml")
```
Comment (kaustubhgupta) [inline Face-Detection/v3/face_detectv3.py:3]:
```suggestion
face_cascade = cv2.CascadeClassifier("./Face-Detection/v3/haarcascade_frontalface.xml")
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by kaustubhgupta — APPROVED** [review]
@Avishake007 💯

**Review by santushtisharma10 — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@antrikshmisri

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**